### PR TITLE
Brush up e2e tests

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: playwright.config.js [hunk 1]
```diff
@@ -7,13 +7,13 @@ const config = {
...
 
+  reporter: 'html',
   use: {
-    headless: true,
     browserName: 'chromium',
-    launchOptions: {
-      // This bit of delay gives async React time to render
-      slowMo: 100,
-    },
+    viewport: {
```
Comment (pavelfeldman) [inline playwright.config.js:13]: You can specify viewport globally.

File: src/__tests__/dependencies.test.js [hunk 2]
```diff
@@ -61,24 +60,12 @@ const tasks = [
 
 const team = {};
 
-test.use(config);
```
Comment (pavelfeldman) [inline src/__tests__/dependencies.test.js:64]: You don't need to do this.

File: src/__tests__/dependencies.test.js [hunk 3]
```diff
@@ -61,24 +60,12 @@ const tasks = [
...
 test.describe("Dependencies", () => {
-  let page;
-
-  test.beforeEach(async ({ browser }) => {
-    const context = await browser.newContext({
```
Comment (pavelfeldman) [inline src/__tests__/dependencies.test.js:69]: You can specify viewport in config, or per file / describe using test.use(). Never create contexts yourself.

File: src/__tests__/team.test.js [hunk 4]
```diff
@@ -23,45 +22,37 @@ const tasks = [
...
-          name: "Brian",
-        },
-        team: {
-          avatar: null,
-          name: "Unclaimed",
-        },
-      })
-    );
+    await Promise.all([
+      page.waitForResponse("https://avatars.githubusercontent.com/**"),
```
Comment (pavelfeldman) [inline src/__tests__/team.test.js:40]: It wasn't react update, it was this network request that made you do slomo. Never do timeouts :)

**Review by bvaughn — APPROVED** [review]
Thanks! 😁

**Comment by bvaughn** [comment]
Thanks for the PR @pavelfeldman!

I get failures when I run tests from this branch locally though, e.g.
```
  4) src/__tests__/team.test.js:31:3 › Teams › should be editable ==================================

    Timeout of 30000ms exceeded.

    Pending operations:
      - page.waitForSelector at src/__tests__/page-utils.js:32:14

    page.waitForSelector: Target closed
    =========================== logs ===========================
    waiting for selector "canvas" to be visible
    ============================================================

      26 |
      27 |   test.beforeEach(async ({ page }) => {
    > 28 |     await loadData(page, { tasks, team });
         |     ^
      29 |   });
      30 |
      31 |   test("should be editable", async ({ page }) => {

        at loadData (/Users/bvaughn/Documents/git/planner/src/__tests__/page-utils.js:32:14)
        at /Users/bvaughn/Documents/git/planner/src/__tests__/team.test.js:28:5
```

Back to regular work today so I won't be able to dig into this for a bit. I'll keep you posted though

**Comment by bvaughn** [comment]
Ugh. Please ignore my previous message. I had something else running on port 3000.

File: src/__tests__/team.test.js (hunk 4 above)
Comment (bvaughn) [reply src/__tests__/team.test.js:40]: Seeing some unexpected weirdness locally from this change:
```
Timeout of 30000ms exceeded.
page.waitForResponse: Page closed
=========================== logs ===========================
waiting for response "https://avatars.githubusercontent.com/**"
============================================================

  37 |     // Change team configuration.
  38 |     await Promise.all([
> 39 |       page.waitForResponse("https://avatars.githubusercontent.com/**"),
     |            ^
  40 |       setEditorText(
  41 |         page,
  42 |         "team",

    at /Users/bvaughn/Documents/git/planner/src/__tests__/team.test.js:39:12
```

Interestingly enough, if I run tests as headfull (no slowMo) they're okay, and if I replace `page.waitForResponse` with `page.waitForRequest` - they're okay to (although the snapshots fail obviously).

File: src/__tests__/dependencies.test.js (hunk 3 above)
Comment (pavelfeldman) [reply src/__tests__/dependencies.test.js:69]: When you let testing harness create contexts for you, you get the best treatment with the tracing and other post-mortem features.

**Pull request merged**